sys/mips/mips/freebsd64_machdep.c

```diff
@@ -247,7 +248,8 @@ freebsd64_set_mcontext(struct thread *td, mcontext64_t *mcp)
 	}
 
 	mc.mc_onstack = mcp->mc_onstack;
-	mc.mc_pc = mcp->mc_pc;
+	mc.mc_cheriframe.cf_pcc = update_pcc_offset(mc.mc_cheriframe.cf_pcc,
```

Comment:
I think update_pcc_offset still KASSERTs when creating invalid caps? We should probably add a non-asserting version? Or if it's now only used for userspace PCC values we can remove the KASSERT.